vhpc_toolkit/get_objs.py

```diff
@@ -281,28 +291,40 @@ def get_vm(self, vm_name, _exit=True):
             else:
                 return None
 
-    def get_network(self, network_name, _exit=True):
+    def get_network(self, network_name, dvs_name=None, _exit=True):
```

Comment:
`network_name` is port group?
Besides, user can might use svs

Reply:
This function should use `port_group` to check what the pg is belonged to, i.e., a dvs or svs.

Reply:
pg can also belongs to a svs